extensions/landing-zone/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "ibmcloud_api_key" {
 variable "region" {
   type        = string
   description = "Region to provision all resources created by this solution"
```

Comment:
Hmm, isn't this supposed to be the region of the cluster? Is there any way to pragmatically get that from cluster ID (maybe not if data lookup requires name)

Reply:
Yes, fixing the description.
data lookup supports ID and so the region could be extracted from the cluster CRN through the datasource ibm_container_vpc_cluste, but unfortunately it needs the region for the ibm provider configuration